### PR TITLE
Added client functionality

### PR DESCRIPTION
**Pull request by nikolarobottesla**
Things I would like to change/fix but I don't know how because I'm new to ansible:
1. I want to make it an option to use the file or a variable for the keys.  Right now the client config only uses a variable and the server only allows a file.  ( I could do this by adding 2 more cli-config files and an associated variable but there is probably a better way?)

2.  The config doesn't work right after the install. In particular it's the edgeos_facts. So I currently run them seperately.

I would like to do these before the merge

### CODE REVIEW

**Comment by nikolarobottesla** [comment]
@holmboe FYI

**Comment by holmboe** [comment]
An observation: at first glance this seem to close #2 as well.

**Comment by nikolarobottesla** [comment]
yes, I saw that issue and addressed it as well

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Role Variables
 Dependencies
 ------------
 
-None
+`ansible-galaxy collection install community.network`
```
Comment (holmboe) [inline README.md]: How about using a [requirements.yaml file](https://galaxy.ansible.com/docs/using/installing.html#installing-multiple-roles-from-a-file) and typing this in the README:

```suggestion
Install using:

- `ansible-galaxy -r requirements.yaml install`
```

Comment (holmboe) [reply README.md]: This is a better documentation of using collection in `requirements.yaml`: https://docs.ansible.com/ansible/latest/user_guide/collections_using.html#install-multiple-collections-with-a-requirements-file

**Comment by holmboe** [comment]
@nikolarobottesla I understand it that you need to create a file named `requirements.yml` with the following content to get this to do what you want:

```
---
collections:
- community.network
```

Disclaimer: I haven't tested the whole scenario myself using your branch so feel free to verify that it is working as intended.

**Comment by nikolarobottesla** [comment]
I verified the requirements install and the client configuration works.  I went a different direction with my wireguard infrastructure and probably won't be using this so I'm not going to try to fix the things I mentioned in the beginning.  I think it is ready to merge now.

**Comment by holmboe** [comment]
@nikolarobottesla Thanks for the update, merged and ready. 👏

**Pull request merged**